composer.json

```
@@ -39,8 +39,9 @@
     "require": {
         "php": ">=7.1",
         "symfony/css-selector": "^4.4|^5.1",
-        "symfony/dom-crawler": "^4.4|^5.1",
         "symfony/dependency-injection": "^4.4|^5.1",
+        "symfony/dom-crawler": "^4.4|^5.1",
+        "symfony/http-client": "^5.2",
```

```suggestion
        "symfony/http-client": "^4.4|^5.1",
```

(perhaps we'll juste require the contract interfaces actually https://github.com/symfony/http-client-contracts, and require-dev `symfony/http-client` which is an implementation, so we have one for testing purposes)

---

Les changements ds le composer.json du commit https://github.com/Elao/seo-tool/pull/16/commits/083640777261ec2f3e84cda40ac88c17e9855113 devraient être revert